syncstorage/db.go

```diff
@@ -52,6 +52,8 @@ const (
 	// The default TTL is to never expire. Use 100 years
 	// which should be enough (in milliseconds)
 	DEFAULT_BSO_TTL = 100 * 365 * 24 * 60 * 60 * 1000
+
+	STORAGE_LAST_MODIFIED = "Storage Last Modified"
```

Comment:
Oh it is actually the name of the key that stores the storage last_modified value.